Permissions Issue

## Expected Behavior
Provide stats for user from PlexPy.

## Actual Behavior
```
undefined:1
undefined
^

SyntaxError: Unexpected token u in JSON at position 0
    at JSON.parse (<anonymous>)
    at Request._callback (/opt/MediaButler/util/plexpy/getUserId.js:12:30)
    at self.callback (/opt/MediaButler/node_modules/request/request.js:186:22)
    at emitOne (events.js:116:13)
    at Request.emit (events.js:211:7)
    at Request.onRequestError (/opt/MediaButler/node_modules/request/request.js:878:8)
    at emitOne (events.js:116:13)
    at ClientRequest.emit (events.js:211:7)
    at Socket.socketErrorListener (_http_client.js:387:9)
    at emitOne (events.js:116:13)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! media-butler@0.0.1 start: `node app.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the media-butler@0.0.1 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

┌──────────────────────────────────────────────┐
│           npm update check failed            │
│     Try running with sudo or get access      │
│     to the local update config store via     │
│ sudo chown -R $USER:$(id -gn $USER) /.config │
└──────────────────────────────────────────────┘
```

## Steps to Reproduce the Problem

  1. `!stats tronyx`

## Specifications

  - Version: Latest
  - Platform: Docker
  - Subsystem: Your mom

--- a/.gitignore
+++ b/.gitignore
@@ -4,3 +4,4 @@ node_modules/*
 
 settings.json
 apiauth.json
+settings.sqlite

--- a/app.js
+++ b/app.js
@@ -47,9 +47,10 @@ client.reload = (command) => {
 
 client.elevation = (message) => {
   let permlvl = 0;
-  if (message.channel.permissionsFor(message.member).hasPermission("MANAGE_MESSAGES")) permlvl = 2;
+  if (message.member.hasPermission("MANAGE_MESSAGES")) permlvl = 2;
   if (message.member.hasPermission("ADMINISTRATOR")) permlvl = 3;
-  if (message.author.id === message.guild.ownerid) permlvl = 4;
+  if (message.author.id == message.guild.ownerID) permlvl = 4;
   return permlvl;
 };
+
 client.login(settings.token);

